Conjunto de alterações 3

BranBadures opened the pull request:
-Adicionada funcionalidade de pesquisa na STL e SAI

BranBadures committed:
Conjunto de alterações 3
-Adicionada funcionalidade de pesquisa na STL e SAI

BranBadures committed:
Alteração na amostragem do status
-Na linha 14 da show.blade havia um if que estava apenas com um sinal de =, assim invés de comparar, ele estava fazendo uma atribuição

ricardfo commented:
Eliminar a repetição de código:

Crie uma nova classe na pasta Utils com a função que retorna as quantidades. Nesta função, que recebe como parâmetro os itens, não precisa ser clonado. Utilize a própria variável para fazer o filtro e a contagem.

BranBadures committed:
Atualização do conjunto 3
-Removido o campo Status da form.blade
-Removida classe quantidades da SaiController e StlController
-Adicionada classe quantidades na Utils/Util
-Adicionada chamada do método quantidades na SaiController e StlController
